assets/app/features/upgrade/i18n/en.json

```diff
@@ -32,50 +32,50 @@
                 },
                 "mode": {
                     "title": "Upgrade Mode",
-                    "description": "After all mandatory checks are successful the upgrade mode will be shown here...",
+                    "description": "After all mandatory checks are successful the upgrade mode of your cloud will be shown here...",
                     "normal": {
-                        "title": "Your cloud is now prepared to begin the upgrade in normal mode.",
+                        "title": "Your cloud is now prepared to begin the upgrade process in normal mode.",
                         "button": "Proceed in Normal Mode",
-                        "description1": "In normal mode, each node will be rebooted during the upgrade process, including the workloads running in them.",
-                        "description2": "In order to proceed in non-disruptive mode your cloud needs to have the pacemaker barclamp correctly configured and applied amd at least one free node as large or larger than any other node in your clouds compute plane.",
+                        "description-title": "IMPORTANT!",
+                        "description1": "In normal mode, the workloads running in each node will be live-migrated to another node. Note that you will lose connection to your VMs when upgrading the control node(s).",
```

Comment:
We actually still have not decided what the disruptive mode will do. I hope this is correct, but it may need another change later.